dali/pipeline/operators/reader/parser/recordio_parser.h

```diff
@@ -70,14 +70,25 @@ class RecordIOParser : public Parser<Tensor<CPUBackend>> {
     ImageRecordIOHeader hdr;
     ReadSingle(&input, &hdr);
 
-    o_label->Resize({1});
-    o_label->mutable_data<float>()[0] = hdr.label;
+    if (hdr.flag == 0) {
+      o_label->Resize({1});
+      o_label->mutable_data<float>()[0] = hdr.label;
+    } else {
+      o_label->Resize({hdr.flag});
+      o_label->mutable_data<float>();
```

Comment:
What it does?

Reply:
`flag` in the header tells how many labels are encoded in this record. calling mutable_data allocates memory if it's not yet allocated. It is not strictly needed here since we do call it again just before we put stuff into o_label.